bedrock/styleguide/templates/styleguide/identity/webmaker-branding.html

```diff
@@ -27,23 +27,23 @@
     <li>use our logos to, or in connection with, content that disparages us or sullies our reputation</li>
   </ul>
   <div class="guidelines-logo">
-    <img src="{{ static('img/styleguide/identity/webmaker/logo.png') }}" alt="Logo" />
+    {{ high_res_img('styleguide/identity/webmaker/logo.png', {'alt': 'Logo'}) }}
```

Comment:
Need a `height` and `width` set so the high res version is constrained to the regular res dimensions.